mentat/app.py

```diff
@@ -57,7 +57,7 @@ def run(paths: Iterable[str], exclude_paths: Optional[Iterable[str]] = None):
     try:
         loop(paths, exclude_paths, cost_tracker)
     except (EOFError, KeyboardInterrupt) as e:
-        print(e)
+        cprint("\n" + str(e), color="red")
```

Comment:
? Why is this here? All it does for me is print a lot of newlines

Reply:
If the error has a message (like the new ones I introduced) then it prints it here - you shouldn't be getting more than 2 new lines even if there error has no message, right?

The "\n" is there to separate the error from the file tree

Reply:
But the error is EOF or KeyboardInterrupt; why would it have a message?